<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nRegionPath</class>
 <widget class="QMainWindow" name="nRegionPath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>161</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Region path</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="defaultValue">
      <property name="neutrinoSave" stdset="0">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>NaN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Min</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Max</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mean</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="doIt">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Extract</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="doMask">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Creates a 0/1 mask from the curve</string>
      </property>
      <property name="text">
       <string>Generate Mask</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QComboBox" name="image">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="neutrinoImage" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="crop">
      <property name="text">
       <string>Crop</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="negative">
      <property name="text">
       <string>Negative</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadPref"/>
   <addaction name="actionSavePref"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
   <addaction name="actionBezier"/>
  </widget>
  <action name="actionLoadPref">
   <property name="icon">
    <iconset>
     <normaloff>:icons//loadPref.png</normaloff>:icons//loadPref.png</iconset>
   </property>
   <property name="text">
    <string>loadPref</string>
   </property>
   <property name="toolTip">
    <string>Load Preferences</string>
   </property>
  </action>
  <action name="actionSavePref">
   <property name="icon">
    <iconset>
     <normaloff>:icons//savePref.png</normaloff>:icons//savePref.png</iconset>
   </property>
   <property name="text">
    <string>savePref</string>
   </property>
   <property name="toolTip">
    <string>Save Preferences</string>
   </property>
  </action>
  <action name="actionBezier">
   <property name="icon">
    <iconset>
     <normaloff>:icons//bezier.png</normaloff>:icons//bezier.png</iconset>
   </property>
   <property name="text">
    <string>bezier</string>
   </property>
   <property name="toolTip">
    <string>Switch Bezier</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset>
     <normaloff>:icons//line.png</normaloff>:icons//line.png</iconset>
   </property>
   <property name="text">
    <string>line</string>
   </property>
   <property name="toolTip">
    <string>Show axis window</string>
   </property>
  </action>
  <action name="actionFlipline">
   <property name="icon">
    <iconset>
     <normaloff>:icons//flipline.png</normaloff>:icons//flipline.png</iconset>
   </property>
   <property name="text">
    <string>flipline</string>
   </property>
   <property name="toolTip">
    <string>Flip Monotone</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/neutrino.qrc"/>
 </resources>
 <connections/>
</ui>
